cpp/src/io/parquet/reader_impl.cu

```diff
@@ -126,9 +128,9 @@ std::tuple<int32_t, int32_t, int8_t> conversion_info(type_id column_type_id,
 {
   int32_t type_width = (physical == parquet::FIXED_LEN_BYTE_ARRAY) ? length : 0;
   int32_t clock_rate = 0;
-  if (column_type_id == type_id::INT8) {
+  if (column_type_id == type_id::INT8 or column_type_id == type_id::UINT8) {
```

Comment:
I suppose this could use `cudf::size_of` (in which case adding a `cudf::is_integer` trait would be nice). I'm not sure why this only returns the width of integer types though?
```cpp
if (cudf::is_numeric(data_type{column_type_id}) and not cudf::is_floating_point(data_type{column_type_id})) { // could be cudf::is_integer instead
  type_width = cudf::size_of(data_type{column_type_id});
}

Reply:
IIRC, the type_width is only nonzero if there is a size conversion going on, such as I32->{I8|U8|I16|U16}, or string->hash32 (but iirc, it is otherwise just ignored, so might be ok to just set the column dtype size for non-string types, though there might be other exceptions besides strings like decimals or INT96)

Reply:
As @OlivierNV said, this is only needed for logical types that are stored using a different physical type in parquet. https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#numeric-types.